bundle/libraries/libraries.go

```diff
@@ -139,11 +140,18 @@ func isLocalLibrary(library *compute.Library) bool {
 		return false
 	}
 
-	return !isDbfsPath(path) && !isWorkspacePath(path)
-}
+	url, err := url.Parse(path)
+	if err != nil {
+		return true
```

Comment:
In what cases does it fail to parse?

I expected this to have to return `!isWorkspacePath(path)` for regular local paths.

Reply:
@pietern it fails primarily when url is malformed https://cs.opensource.google/go/go/+/refs/tags/go1.21.0:src/net/url/url.go;l=499

but agree that it should return !isWorkspacePath(path) in this case